src/main/scala/gitbucket/core/view/helpers.scala

```diff
@@ -463,6 +463,16 @@ object helpers extends AvatarImageProvider with LinkConverter with RequestCache
     Html(sb.toString())
   }
 
+  /**
+   * Count conversations(commit-comment or issue-comment action).
+   *
+   * @param comments all comments(includes actions)
+   * @return conversation count
+   */
+  def countConversation(comments: Seq[Comment]): Int = {
+    comments.count(c => c.isInstanceOf[CommitComments] || c.asInstanceOf[IssueComment].action.endsWith("comment"))
+  }
```

Comment:
I'm not sure whether we should add this method to helpers because this isn't a generic method for view construction. What do you think about following alternatives:

- Write inline this counting or define the method in the view
- Count in the controller and pass it to the view

Reply:
@takezoe Thanks for your review!
I've changed to count conversations at the view of pull request screen.